docs/use_case_examples.md

```diff
@@ -0,0 +1,36 @@
+# Use Case examples.
```

Comment:
Only missing a link to this file in the main README. After that we can probably accept this PR